<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalSettings</class>
 <widget class="QWidget" name="LocalSettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocalSettings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="DeviceTypeGroupBox">
     <property name="title">
      <string>Device Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="DeviceTypeComboBox">
        <item>
         <property name="text">
          <string>M5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MPC04</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="ConfigFileGroupBox">
     <property name="title">
      <string>Config File</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="ConfigFileLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="ConfigFilePushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="LogFilePathGroupBox">
     <property name="title">
      <string>LogFile Path</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLineEdit" name="LogFilePathLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="LogFilePathPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="TestEngineGroupBox">
     <property name="title">
      <string>TestEngine</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="TestEngineNumberLabel">
        <property name="text">
         <string>Engine Numbers:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="TestEngineNumberSpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="TestEngineTableWidget">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Engine Index</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Device SerialNumber</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="ModifySerialNumberPushButton">
        <property name="text">
         <string>Modify SerialNumber</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
